armi/reactor/parameters/parameterDefinitions.py

```diff
@@ -224,6 +224,31 @@ def setParameter(selfObj, value):
     return setParameter
 
 
+def isNumpyArray32Bit(paramStr: str):
```

Comment:
[`numpy.int32`](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int32) is also a thing. We usually use floats so _probably_ not likely to have any disambiguation. and the docstring says it's a 32bit float.

But for extra explicitness, non-blocking recommend to
```suggestion
def isNumpyF32Array(paramStr: str):
```